city_scrapers/spiders/cuya_audit.py

```diff
@@ -1,27 +1,17 @@
-import re
-
 from city_scrapers_core.constants import ADVISORY_COMMITTEE
 from city_scrapers_core.spiders import CityScrapersSpider
 
-from city_scrapers.mixins import CuyaCountyMixin
+from city_scrapers.mixins import CuyaCountyMixin2
 
 
-class CuyaAuditSpider(CuyaCountyMixin, CityScrapersSpider):
+class CuyaAuditSpider(CuyaCountyMixin2, CityScrapersSpider):
     name = "cuya_audit"
     agency = "Cuyahoga County Audit Committee"
-    start_urls = ["http://bc.cuyahogacounty.us/en-US/Audit-Committee.aspx"]
+    start_urls = [
+        "https://cuyahogacounty.gov/boards-and-commissions/board-details/external/audit-committee"  # noqa
+    ]
     classification = ADVISORY_COMMITTEE
     location = {
         "name": "County Headquarters, 4-407 Conference Room B",
         "address": "2079 East 9th St Cleveland, OH 44115",
     }
```

Comment:
- drop: location


- Adobe
- Microsoft office
- Share feedbakc


Small request here, Hails. Could you please delete this class variable? In this case, CuyaCountyMixin2 is handling the [parsing](https://github.com/City-Bureau/city-scrapers-cle/blob/62fb1f3722fd2f82c90713f9cc95a5241bb80c91/city_scrapers/mixins/cuya_county2.py#L106) of location info from the page – so this variable is essentially unused.

Just in case there's any confusion here: I know in the [city-scrapers-indianapolis PR](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14#discussion_r1452766903) we discussed treating location info as a constant via class variables. I think that approach makes sense for certain agencies that don't display location information on their meeting detail pages and we generally know a meeting is always going to be in the same place. In cases where location information is reliably displayed on meeting detail pages – which appears to be the case with Cuyahoga County agencies – I think it makes sense to parse it from the page. Since meeting venues can occasionally shift around (especially between years), I think this increases the chance we're scraping good data longterm.